src/saveload/vehicle_sl.cpp

```diff
@@ -759,6 +759,7 @@ const SaveLoad *GetVehicleDescription(VehicleType vt)
 		     SLE_VAR(RoadVehicle, overtaking_ctr,       SLE_UINT8),
 		     SLE_VAR(RoadVehicle, crashed_ctr,          SLE_UINT16),
 		     SLE_VAR(RoadVehicle, reverse_ctr,          SLE_UINT8),
+		SLE_CONDDEQUE(RoadVehicle, path,                SLE_UINT8,                  SLV_ROADVEH_PATH_CACHE, SL_MAX_VERSION),
```

Comment:
alignment :)

Reply:
Looks intentional to me, `SLE_CONDDEQUE` is longer than any of the other so if it were to be aligned further out it'd have fewer tabs at the start of the line.

Reply:
Yeah I *could* realign every SLE_VAR :/

Reply:
Actually the `SLE_UINT8` is in fact not aligned.